services/web/server/src/simcore_service_webserver/groups_handlers.py

```diff
@@ -59,16 +60,31 @@ async def list_groups(request: web.Request):
 
     List of the groups I belonged to
     """
+    from .products import Product, get_current_product
```

Comment:
why is this import here?

Reply:
this is because the routes are collected using an old map function ``get_handlers_from_namespace(groups_handlers)`` that "thinks" that ``get_current_product`` is a handler. All theis function is deprecated and will go away when we have removed opeanpi-core